packages/discovery-provider/ddl/migrations/0044_set_release_to_created_where_null.sql

```diff
@@ -0,0 +1,7 @@
+begin;
+
+UPDATE tracks
+SET release_date = created_at
+WHERE release_date IS NULL;
+
```

Comment:
is it easy to modify this column to not null?

Reply:
hmm like alter release_date to add a not null constraint? i could do that

Reply:
my b CI not passing cuz of this. i'd say either remove or CURRENT_TIMESTAMP